src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -478,6 +475,11 @@ internal Accessibility LocalAccessibility
         /// </remarks>
         internal bool ContainsFieldKeyword => _containsFieldKeyword;
 
+        /// <summary>
+        /// Indicates whether this accessor has no body in source and a body will be synthesized by the compiler.
+        /// </summary>
+        internal bool AccessorBodyShouldBeSynthesized => _accessorBodyShouldBeSynthesized;
```

Comment:
>Accessor [](http://example.com/codeflow?start=22&length=8)

I think we can drop the "Accessor" prefix from the name of the property and the backing field.